articles/multifactor-authentication/developer/step-up-with-acr.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
I'd also say that if you want to request step up in place you should add a nonce and check it afterwards, or (less secure) at least check the token iat for it to be recent; otherwise id tokens last a day and could be re-played to you at any time if a third party captures it (many people put them in local storage, which is pretty easy to access)